aim/tests/unit/tools/cli/test_manager.py

```diff
@@ -389,6 +390,52 @@ def test_sync_state_recover(self):
             # Items are updated
             self.assertEqual(4, up.call_count)
 
+    def test_tag_list_command(self):
+        self.mgr.create(self.ctx, resource.Tenant(name='tn1'))
+        mo_dict = {'tagInst': {'attributes':
+                   {'dn': 'uni/tn-tn1/tag-openstack_aid'}}}
+        mock_mo = mock.MagicMock()
+        mock_mo.__getitem__.side_effect = mo_dict.__getitem__
+        MO_LIST = [mock_mo]
```

Comment:
We can't just do "MO_LIST = [mo_dict]" here?

Reply:
If I change it to use the mock on click.echo, then yes we could. I did it on the mock_mo so that I could verify that calls.